libc/test/src/unistd/readlink_test.cpp

```diff
@@ -30,17 +31,20 @@ TEST(LlvmLibcReadlinkTest, CreateAndUnlink) {
   //   3. Cleanup the symlink created in step #1.
   ASSERT_THAT(LIBC_NAMESPACE::symlink(LINK_VAL, LINK), Succeeds(0));
 
-  char buf[sizeof(LINK_VAL)];
```

Comment:
Why don't we just hardcoded some big enough buffer?  Something like
```
char buf[256];
```

Reply:
done